src/State/Exporter.php

```
@@ -0,0 +1,48 @@
+        $ref = new ReflectionClass($value);
+        $id = $ref->getMethod('getIdentity')->invoke($value);
+
+        return parent::exportValue($serializer, $field, $id, $runningValue);
+    }
+
+    public function canExport(Field $field, mixed $value, string $format): bool
+    {
+        return $value instanceof Record;
+    }
+
+    public function importValue(Deserializer $deserializer, Field $field, mixed $source): mixed
```

PHPMD (2.15.0)

Avoid unused parameters such as '$deserializer'. (Unused Code Rules)

(UnusedFormalParameter)